Elevel/Application/Features/ReportCommands/CreateReportCommand.cs

```diff
@@ -96,6 +103,12 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     }
 
                     report.CreatorId = question.CreatorId;
+                    
+                    userEmail = (await _userManager.Users.FirstOrDefaultAsync(x => x.Id == question.CreatorId)).Email;
+                    if (string.IsNullOrWhiteSpace(userEmail))
+                    {
+                        throw new NotFoundException($"User with {request.UserId}");
```

Comment:
but your exception message is not correct. Could you change message about empty or null email?

Reply:
Done